bin/sdk-cli.js

```diff
@@ -72,6 +72,10 @@ yargs
 				description: 'Whether to watch for changes and automatically rebuild.',
 				type: 'boolean',
 			},
+			'namespace': {
+				description: 'Wheater CSS be namespaced.',
```

Comment:
Typo here: should be `whether`. Also, I think it's missing a "should" right before "be".